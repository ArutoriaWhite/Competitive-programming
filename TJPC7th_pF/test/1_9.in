73654984 2
54756005 0
70641730 0
