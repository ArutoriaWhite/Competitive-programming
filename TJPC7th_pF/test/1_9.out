70641730 15885725
